packages/main/src/Label.js

```diff
@@ -139,6 +144,13 @@ class Label extends UI5Element {
 		};
 	}
 
+	get tabIndex() {
```

Comment:
we can remove this. instead of it we can call _tabIndex in the Label.hbs since it doesn't have defaultValue it will return undefined if not set

Reply:
Good point... if I declare defaultValue=undefined, then the tabIndex getter becomes redundant
Refactored